### PR TITLE
Add 2.1 blog post

### PR DESCRIPTION
**Pull request by sosukesuzuki**
There's still todo for 2.1 milestone, but I create PR for a quick review and release.  This PR is still draft but reviews welcome! I'm not good at English, so it would be helpful if you could review English.

## TODO

- [x] Update contents when we've completed 2.1 milestone.
  - ~~If #8063 is completed, I'll add it to `Highlights`~~
- [x]  Remove all `pr-xxxx.md` from `/changelog_unreleased`
- [x]  Rewrite `// Prettier stable` => `// Prettier 2.0`, `// Prettier master` -> `// Prettier 2.1`
  - write a script on other PR (#8984)
- [x] Update filename

https://deploy-preview-8932--prettier.netlify.app/blog/2020/08/24/2.1.0.html

### CODE REVIEW

File: website/blog/2020-08-04-2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,2241 @@
...
+  .should("have.css", "color", "#aaa");
+
+// Prettier stable
+cy.get(".ready").should("have.text", "FOO").should("have.css", "color", "#aaa");
+
+// Prettier master
+cy.get(".ready")
+  .should("have.text", "FOO")
+  .should("have.css", "color", "#aaa");
+```
```
Comment (sosukesuzuki) [inline website/blog/2020-08-04-2.1.0.md:144]: If #8495 is merged, we'll remove this

File: cspell.json [hunk 2]
```diff
@@ -327,6 +327,8 @@
         "softline",
         "softlines",
         "Sorin",
+        "Sosuke",
+        "sosukesuzuki",
```
Comment (fisker) [inline cspell.json:331]: Can we add patter to ignore `author: ".*？"`？

**Comment by fisker** [comment]
#8410 codeblock not right
#7592 #7844 codeblock not highlighted

File: website/blog/2020-08-04-2.1.0.md [hunk 3]
```diff
@@ -0,0 +1,2241 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.1: new --embedded-language-formatting option and new JavaScript/TypeScript features!"
+---
+
+This release adds new `--embedded-language-formatting` option, support new JavaScript/TypeScript features and a lot of bugfixes and improvements!
```
Comment (brodybits) [inline website/blog/2020-08-04-2.1.0.md:7]:
```suggestion
This release adds a new `--embedded-language-formatting` option, supports new JavaScript/TypeScript features, and includes many bug fixes and improvements!
```

File: website/blog/2020-08-04-2.1.0.md [hunk 4]
```diff
@@ -0,0 +1,2241 @@
...
+// Prettier stable
+for   (const  element of list) { /* ... */ }
+
+// Prettier master
+for (const element of list) {
+  /* ... */
+}
+```
+
+#### Support Private Fields in `in` ([#8431](https://github.com/prettier/prettier/pull/8431) by [@sosukesuzuki](https://github.com/sosukesuzuki))
```
Comment (brodybits) [inline website/blog/2020-08-04-2.1.0.md:424]: only on Babel parsers, right?

(I did just raise PR #8942 to run the test on all Babel parsers.)

File: website/blog/2020-08-04-2.1.0.md [hunk 5]
```diff
@@ -0,0 +1,2241 @@
...
+
+// Prettier master
+type Range = [start: number, end: number];
+```
+
+## Other changes
+
+### JavaScript
+
+#### Support the F# and Smart pipeline operator proposals ([#6319](https://github.com/prettier/prettier/pull/6319) by [@sosukesuzuki](https://github.com/sosukesuzuki), [@thorn0](https://github.com/thorn0), [#7979](https://github.com/prettier/prettier/pull/7979) by [@sosukesuzuki](https://github.com/sosukesuzuki))
```
[the remark this answers is not in the document]
Comment (sosukesuzuki) [reply website/blog/2020-08-04-2.1.0.md:84]: Yes!

File: website/blog/2020-08-04-2.1.0.md (hunk 4 above)
Comment (sosukesuzuki) [reply website/blog/2020-08-04-2.1.0.md:424]: Yes, works on only babel

File: cspell.json (hunk 2 above)
Comment (sosukesuzuki) [reply cspell.json]: done: 36cce5c
Please review because I'm not familiar with regexp

File: cspell.json [hunk 6]
```diff
@@ -402,7 +401,8 @@
         "\\[`\\w+`\\]",
         "ve{2,}r{2,}y",
         "ve+r+y+long\\w*",
-        "\\/.*?\\/"
+        "\\/.*?\\/",
+        "\"?author.*\"?: \".*\""
```
Comment (fisker) [inline cspell.json:405]:
```suggestion
        "author: \".*?\"",
        "authorURL: \".*?\"",
        "\"author\": \".*?\""
```

Let's use separate regrex, more clear

File: website/blog/2020-08-04-2.1.0.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ authorURL: "https://github.com/sosukesuzuki"
 title: "Prettier 2.1: new --embedded-language-formatting option and new JavaScript/TypeScript features!"
 ---
 
-This release adds new `--embedded-language-formatting` option, support new JavaScript/TypeScript features and a lot of bugfixes and improvements!
+This release adds a new `--embedded-language-formatting` option, supports new JavaScript/TypeScript features, and includes many bug fixes and improvements!
```
Comment (fisker) [inline website/blog/2020-08-04-2.1.0.md:7]: Don't you want update the original changelog? You might want run the `draft-blog-post.js` script again.

Comment (fisker) [reply website/blog/2020-08-04-2.1.0.md:7]: Sorry, I mean other changes from changelog, not this one.

File: website/blog/2020-08-21-2.1.0.md [hunk 8]
```diff
@@ -42,37 +42,56 @@ I am expecting this to come out exactly like it went in.
...
+> 1 | type Range = [start: number, end: number];
 
 // Prettier master
-try {
-} catch (e: any) {}
+type Range = [start: number, end: number];
 ```
 
-#### Support `Labeled Tuple Elements` ([#8885](https://github.com/prettier/prettier/pull/8885) by [@fisker](https://github.com/fisker), [#8982](https://github.com/prettier/prettier/pull/8982) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+#### Short-Circuiting Assignment Operators ([#8982](https://github.com/prettier/prettier/pull/8982) by [@sosukesuzuki](https://github.com/sosukesuzuki))
```
Comment (sosukesuzuki) [inline website/blog/2020-08-21-2.1.0.md:66]: I added about `Short-Circuiting Assignment Operators` here https://github.com/prettier/prettier/pull/8982#issuecomment-678085919

**Comment by fisker** [comment]
@sosukesuzuki https://github.com/prettier/prettier/pull/8932#issuecomment-671078615

**Comment by vjeux** [comment]
I have issues connecting to my account right now. @j-f1, @duailibe, @suchipi could one of you help? Thanks!

**Comment by thorn0** [comment]
@sosukesuzuki I added you.

**Comment by sosukesuzuki** [comment]
@thorn0 @fisker Should we merge this before running release script?

**Comment by fisker** [comment]
I don't know, but according to https://github.com/prettier/prettier/issues/8882, merge after release.

**Comment by fisker** [comment]
So the script you wrote to replace prettier version won't work? You can replace it manually, let's improve it later.

**Comment by sosukesuzuki** [comment]
Yes, I was worried about that too. In this release I'll replace it manually.

**Comment by AlCalzone** [comment]
Maybe I'm blind, but whats the difference under "Fix unstable comments in binary expressions"?
```js
// Prettier 2.0 (first output)
Math.min(
  /* foo */
  document.body.scrollHeight -
    (window.scrollY + window.innerHeight) -
    devsite_footer_height,
  0
);

// Prettier 2.0 (second output)
Math.min(
  /* foo */
  document.body.scrollHeight -
    (window.scrollY + window.innerHeight) -
    devsite_footer_height,
  0
);
```
looks identical to me.

**Comment by fisker** [comment]
@AlCalzone Thank you, I fixed it https://github.com/prettier/prettier/commit/2fe04158a7e9189d270946dfb36181579722b711

**Comment by vjeux** [comment]
Congratz everyone on the release!

**Pull request merged**